package/crds/iam.aws.upbound.io_users.yaml

```diff
@@ -103,12 +102,11 @@ spec:
                 properties:
                   forceDestroy:
                     description: when destroying this user, destroy even if it has
-                      non-Upbound official provider-managed iam access keys, login
-                      profile or mfa devices. without force_destroy a user with non-Upbound
-                      official provider-managed access keys and login profile will
-                      fail to be destroyed. delete user even if it has non-Upbound
-                      official provider-managed iam access keys, login profile or
-                      mfa devices
+                      non-provider-managed iam access keys, login profile or mfa devices.
```

Comment:
Should we make this change here? Isn't provider-aws still the official provider of Upbound?

Reply:
My device cannot load the page :) also, there is a similar situation in the other two CRDs.

Reply:
Hi @turkenf,
We are planning to address this in a follow-up PR: https://github.com/crossplane/upjet/pull/279#discussion_r1354326038